kyo-bench/src/main/scala/kyo/bench/DeepBindBench.scala

```diff
@@ -7,8 +7,8 @@ class DeepBindBench extends Bench.SyncAndFork(()):
     def kyoBench() =
         import kyo.*
 
-        def loop(i: Int): Unit < IOs =
-            IOs {
```

Comment:
`IOs` was used to ensure stack safety via suspension. Since Kyo's execution is stack-safe by default now, the suspension isn't necessary anymore, which makes Kyo's impl of the benchmark closer to the others.